lib/msf/core/db_export.rb

```diff
@@ -35,10 +35,10 @@ def host_allowed?(arg)
   def to_pwdump_file(path, &block)
     exporter = Metasploit::Credential::Exporter::Pwdump.new(workspace: workspace)
 
-    File.open(path, 'w') do |file|
+    output_file = File.open(path, 'w') do |file|
       file << exporter.rendered_output
     end
-    true
+    output_file.path
```

Comment:
Minor: Update method comment to reflect change in return value

Reply:
Made this change.